tezos/api/src/ffi.rs

```diff
@@ -744,7 +744,7 @@ impl From<TezosErrorTrace> for FfiJsonEncoderError {
 
 pub type Json = String;
 
-#[derive(Serialize, Deserialize, Debug)]
+#[derive(Clone, Serialize, Deserialize, Debug)]
```

Comment:
@tizoc 
and please remove also the unneeded Clone

Reply:
Ok, all done.